tests/functional/cache_test.c

```diff
@@ -740,9 +741,8 @@ test_reader_thread(void *arg)
    platform_semaphore_init(&params->batch_sema, 0, params->hid);
    for (i = k = 0; i < num_pages; i += READER_BATCH_SIZE) {
       if (params->logger) {
-         platform_throttled_error_log(DEFAULT_THROTTLE_INTERVAL_SEC,
-                                      PLATFORM_CR "test %3lu%% complete",
-                                      i * 100 / num_pages);
+         platform_error_log(PLATFORM_CR "test %3lu%% complete",
```

Comment:
Any reason why this was changed from throttled to just `_error_log()`?

I have issue #336 opened to clean-up `throttled_error_log`. Once that happens, this code will function better.

Did you want to simply move away from throttled to normal error_log here?

Reply:
Since it wasn't implemented, I thought we might as well deprecate it. But that was prior to your issue, so I'm happy to revisit it and possibly restore this code.

I believe this was something only implemented on the kernel side.

Reply:
After reading your later comment and thinking about it, I think it makes sense to leave these. I went ahead and reverted them.